providers/caching/memoryConfig.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const memory = require('./memory')
+
+function serviceFactory() {
+  return memory({ defaultExpirationSeconds: 60 * 60 })
+}
+
+module.exports = serviceFactory
```

Comment:
memory.js can be its own factory. It already is one. Just need to add the default options in memory.js